03.asynchronous/database.js

```diff
@@ -0,0 +1,236 @@
+#! /usr/bin/env node
+
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const database = new sqlite3.Database(":memory:");
+const createtableSQL = `
```

Comment:
単語の区切り的に `createTableSQL` だと思います。

```suggestion
const createTableSQL = `
```